.github/release.config.js

```diff
@@ -6,17 +6,17 @@ module.exports = {
             level: "minor"
         },
         {
-            name: "release/2.*",
+            name: "v2-lts",
             channel: "zowe-v2-lts",
             level: "patch"
         },
         {
-            name: "release/3.*",
-            channel: "latest",
+            name: "v1-lts",
+            channel: "zowe-v1-lts",
             level: "patch"
         },
         {
-            name: "next",
+            name: "release/3.*",
```

Comment:
@traeok I added this for the final pre-release, looking at the comment in our slack from @t1m0thyj isn't the plan to continue v3-lts releases against main.  I may have misunderstood the conversation though...

[convo](https://openmainframeproject.slack.com/archives/G01A99BR6PP/p1726600745676719)

Reply:
Sorry, I should have clarified that the `release/3.x` branch implies any branch that starts with `release/3.` - so if we release 3.1, we can release patches from the `release/3.1` branch without having to "filter out" enhancements that were already pushed into `main` after the 3.1 code freeze. 

As for the final pre-release, I think we can release against `main` - once v3 is released as GA, we can make a branch `release/3.0` (based on the latest changes on `main`). If we need to release a patch for 3.0, we can publish changes that are merged into that branch.

Reply:
I have a pre-release [branch](https://github.com/zowe/zowe-explorer-vscode/tree/release/3.0.0-pre-release) already built off of these changes and a [PR](https://github.com/zowe/zowe-explorer-vscode/tree/v3-final-prerelease) for the final pre-release already created, and we would have to either change the `release/3.*` or the `main` config setting.

Reply:
Oh okay, I see - that's another option we can take, so that sounds good to me. But yes, we'll have to change it regardless after the final pre-release 😁

Reply:
cool, I can create the PR to update the `release/3.*` setting after last pre-release goes out, shooting for EOD tomorrow (9/24).  We will need these changes merged in for this to publish and my follow up PR for GA publishing 😄